BuildResidentialHPXML/measure.rb

```diff
@@ -3223,7 +3223,7 @@ def self.create_schedules(runner, model, epw_file, args)
     return false if not success
 
     # export the schedule
-    args[:schedules_path] = '../schedules.csv'
+    args[:schedules_path] = "../#{File.basename(args[:hpxml_path], '.xml')}.csv"
```

Comment:
How about `"../#{File.basename(args[:hpxml_path], '.xml')}_schedules.csv"`, so it's more obvious what's in the csv file?